src/App.tsx

```diff
@@ -40,9 +42,23 @@ const App = () => {
 
   const dispatch = useDispatch()
 
-  const connectAccount = useCallback(async () => {
+  const switchLedgerType = async (ledgerType: string) => {
+    let ledger
+    switch (ledgerType) {
+      case CosmosNetworkConfig.KEPLR_LEDGER:
+        ledger = await connectKeplrLedger()
+        return ledger
+      case CosmosNetworkConfig.COSMOSTATION_LEDGER:
+        ledger = await connectCosmostationLedger()
+        return ledger
+      default:
+        return { address: '', accountName: '' }
+    }
+  }
+
```

Comment:
I think switchLedgerType() can be a helper function and called when needed, as it is currently duplicated in App.tsx and ConnectWallet.tsx

Reply:
Done